src/components/Article.jsx

```diff
@@ -17,6 +18,27 @@ function Article() {
             })
     }, [item_id]);
 
+    function handleVote(voteChange) {
```

Comment:
Nice use of a reusable function for increasing or decreasing votes here!
Will you disable the buttons once a user has voted?